### ISSUE TITLE
`CleanParam` directive

### ISSUE THREAD

**Issue by ExEr7um** [harlan-zw/nuxt-simple-robots#66]
### Clear and concise description of the problem

I need to use a `CleanParam` directive, that is developed by Yandex and removes unnecessary query-params from URL. You can read more about it [here](https://yandex.com/support/webmaster/robot-workings/clean-param.html).

### Suggested solution

This directive works the same as `Allow` or `Disallow`, so it should be implemented just the same.

### Alternative

Maybe a way to use any custom directive, that will work like `Allow` or `Disallow`, but it can get any custom name.

**Comment by harlan-zw** [comment]
Thanks for the issue! This is available in [4.0.0-rc.12](https://github.com/harlan-zw/nuxt-simple-robots/releases/tag/v4.0.0-rc.12).

I've coded it so you can only use the directive when you are specifically targeting Yandex, which seems to be the guidelines based on their documentation.

So the following robots.txt config would work

```txt
User-agent: *
Disallow: /admin

User-agent: Yandex
Disallow: /admin
Clean-param: s /forum/index.php
Clean-param: s /forum/showthread.php

Sitemap: https://yoast.com/sitemap_index.xml
```

Alternatively, you can configure it like this:

```ts
export define defineNuxtConfig({
  robots: {
    groups: [
      {
        userAgent: '*',
        disallow: '',
      },
      {
        userAgent: 'Yandex',
        cleanParam: ['s /forum/index.php', 's /forum/showthread.php']
      },
    ],
  }
})
```

Let me know if you have issues.

**Comment by ExEr7um** [comment]
Thank you!
